### PR TITLE
Update /builds, add LTS blog post

### PR DESCRIPTION
**Pull request by mixonic**

### CODE REVIEW

File: source/blog/2016-02-19-announcing-embers-first-lts.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+![LTS release timeline](/images/blog/2016-02/Releases@3x.svg)
+
+In reality release dates will drift by several days, however the guidance
+above should provide a sense of scale for the cadence of LTS releases. These
+will happen roughly twice a year.
+
+### The LTS Channel
+
+What makes an LTS release different than an stable Ember release? LTS releases
```
Comment (locks) [inline source/blog/2016-02-19-announcing-embers-first-lts.md]: "an stable"

File: source/blog/2016-02-19-announcing-embers-first-lts.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+come with guarentees about API stability and security:
```
Comment (locks) [inline source/blog/2016-02-19-announcing-embers-first-lts.md]: "guarantees"

File: source/blog/2016-02-19-announcing-embers-first-lts.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+* Security fixes will always be back-ported to the most recent LTS release. [Read
+  more about Ember's security policy](http://emberjs.com/security/).
+* Release notes for each LTS will contain a summary of features and
+  deprecations added since the previous LTS. This will likely consist of a
+  number of links to documentation, guides, and release notes for stable
+  releases.
+* Addons and applications should constrain themselves to using Ember's public
+  API. However, in practice we recognize that experiementation and use of
```
Comment (locks) [inline source/blog/2016-02-19-announcing-embers-first-lts.md]: "experiementation"

**Comment by mixonic** [comment]
Thanks @locks - I also normalized language around "Ember" over "Ember.js"

**Comment by locks** [comment]
:heart_eyes: :+1:

**Comment by rwjblue** [comment]
LGTM

**Pull request merged**